### PR TITLE
Rename seedu.address to seedu.zerotoone

### PR DESCRIPTION
**Pull request by alloystory**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@ bin/
 .classpath
 .project
 .settings/
+.vscode/
```
Comment (jiachen247) [inline .gitignore]: Could you create a new section for vscode please!

**Review by jiachen247 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (alloystory) [reply .gitignore]: noted!

**Review by chishanw — APPROVED** [review]
LGTM!

**Review by jiachen247 — APPROVED** [review]
LGTM

**Pull request merged**